src/main/java/genie/task/Event.java

```diff
@@ -2,20 +2,39 @@
 
 import genie.task.Task;
 
+/**
+ * Deals with creating event tasks.
+ */
 public class Event extends Task {
     protected String from;
```

Comment:
Perhaps more descriptive names can be used for these fields that store dates instead of just "to"and "from"?